elements/lisk-transactions/src/base_transaction.ts

```diff
@@ -221,6 +219,10 @@ export abstract class BaseTransaction {
 		return transaction;
 	}
 
+	public stringify(): string {
```

Comment:
what's the reasoning to add this function?

Reply:
`console.log(new MyTransaction(...).sign('my passphrase').sendable())`

Reply:
you mean `console.log(new MyTransaction(...).sign('my passphrase').stringify())`?
when should it be used like this?

Reply:
Yep, I meant this, before sending :)